### PR TITLE
ESBJAVA-3493 - Warning log message was added for aggregate mediator t…

### PR DESCRIPTION
**Pull request by ravindraranwala**
…imeout scenario.

### CODE REVIEW

**Comment by chanakaudaya** [comment]
Hi Ravindra,

This diff contains unnecessary changes. Please remove all the redundant changes and resend the PR.

File: modules/core/src/main/java/org/apache/synapse/mediators/eip/aggregator/Aggregate.java [hunk 1]
```diff
@@ -19,16 +19,16 @@
...
+import java.util.List;
+import java.util.TimerTask;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
 import org.apache.synapse.MessageContext;
 import org.apache.synapse.SynapseLog;
 import org.apache.synapse.core.SynapseEnvironment;
 import org.apache.synapse.mediators.eip.EIPConstants;
-import org.apache.commons.logging.Log;
```
Comment (chanakaudaya) [inline modules/core/src/main/java/org/apache/synapse/mediators/eip/aggregator/Aggregate.java]: Remove these unnecessary changes.

**Comment by ravindraranwala** [comment]
Already merged the fix via pr https://github.com/wso2/wso2-synapse/pull/250, hence closing this pr which is redundant.
